BrowserKit/Sources/Shared/TimeConstants.swift

```diff
@@ -174,6 +177,11 @@ extension Date {
     public func isWithinLastHour(comparisonDate: Date = Date()) -> Bool {
         return (comparisonDate.lastHour ... comparisonDate).contains(self)
     }
+
+    /// Comparison date is used to control unit tests outcome.
+    public func isWithinLastTwentyFourHours(comparisonDate: Date = Date()) -> Bool {
+        return (comparisonDate.lastTwentyFourHours ... comparisonDate).contains(self)
+    }
```

Comment:
Seems like this isn't used anywhere else but for tests, can we add these as private method to the tests class below and so that we don't have this exposed in our production code?